Lightweight.Dockerfile

```
@@ -21,19 +21,21 @@ RUN apt-get update && \
     git && \
     rm -rf /var/lib/apt/lists/*
 
+# Install uv for faster dependency management
+RUN pip install uv
```

Hadolint (2.14.0)

[warning] Avoid use of cache directory with pip. Use `pip install --no-cache-dir <package>`

(DL3042)